ConsoleApp1/Program.cs

```diff
@@ -1,9 +1,140 @@
-﻿namespace ConsoleApp1
+﻿
+using System.Text.RegularExpressions;
+
+namespace ConsoleApp1
 {
 	internal class Program
 	{
 		static void Main(string[] args)
 		{
+			string a = "hello";
+			string b = "hello,*%";
+			string forSorting = "DCBaz";
+
+			//Compare(a, b);//#1
+
+			int counterForChars, counterForDigits, counterForSpecialChars;
+            Analyze("st1", out counterForChars,  out counterForDigits, out  counterForSpecialChars);//#2
+
+			//Sort(forSorting);//#3
+			//Duplicate("yroma roma rota ta wwy y");//#4 
+        }
+
+		//#1
+        //Compare that will return true if 2 strings are equal,
+		//otherwise false, but do not use build-in method
+
+        public static bool Compare(string a, string b)
+		{
+			if (a.Length != b.Length)
+			{
+				Console.WriteLine($"False, a.length = {a.Length!}, b.Length = {b.Length!}");
+				return false;
+			}
+
+			if( a != b)
+			{
+				Console.WriteLine($"{a} not equal to {b}");
+				return false;
+			}
+
+			Console.WriteLine($"True, {a} equal to {b}");
+			return true;
+		}
+
+		//#2
+		//Analyze that will return number of alphabetic chars in string,
+		//digits and another special characters
+
+		//TODO
+		public static void Analyze(string s, out int counterForChars, out int counterForDigits, out int counterForSpecialChars)
```

Comment:
There is an easier solution to use char.IsXXXXX() methods in the loop of an array of characters